docs/reference/function-configuration/function-configuration-reference.md

```diff
@@ -194,7 +194,7 @@ spec:
 
 ## Function Status (`spec`)
```

Comment:
```suggestion
## Function Status (`status`)
```

Reply:
I've commit one more typo that you pointed :)